core/block_rendering_rewrite/block_render_info.js

```diff
@@ -639,6 +639,12 @@ Blockly.BlockRendering.RenderInfo.prototype.finalize_ = function() {
   this.blockBottom = yCursor;
```

Comment:
Check whether we're still using `blockBottom`.  If we are, your change to the `yCursor` needs to be above this line.  If not, add an item to the gnosis to delete it.

Reply:
Doesn't look like we are using it. Will add a task for cleanup in gnosis.